### PR TITLE
Complete the updates for version 1.1

### PR DESCRIPTION
**Pull request by DrDaveD**
This has a lot more updates to bring the userdocs up to date with apptainer 1.1.0

### CODE REVIEW

File: build_a_container.rst [hunk 1]
```diff
@@ -149,26 +147,25 @@ You can do so with the following command.
...
    compressor available is ``gzip``.
 
 *******************************
  Building encrypted containers
 *******************************
 
 With {Project} it is possible to build and run
-encrypted containers. The containers are decrypted at runtime entirely
+encrypted containers. 
+When running in setuid-mode,
```
Comment (ikaneshiro) [inline build_a_container.rst:167]: We should make it clearer that setuid-mode is the only way to run an encrypted container.

File: definition_files.rst [hunk 2]
```diff
@@ -87,23 +87,22 @@ Preferred bootstrap agents
...
 ======================
 
+-  :ref:`library <build-library-module>` (images hosted on Library API Registries)
 -  :ref:`shub <build-shub>` (images hosted on Singularity Hub)
--  :ref:`localimage <build-localimage>` (images saved on your machine)
 -  :ref:`yum <build-yum>` (yum based systems such as CentOS and
    Scientific Linux)
 -  :ref:`debootstrap <build-debootstrap>` (apt based systems such as
    Debian and Ubuntu)
 -  :ref:`oci <cli-oci-bootstrap-agent>` (bundle compliant with OCI Image
```
Comment (ikaneshiro) [inline definition_files.rst:103]: This one also doesnt have a section to link to

File: index.rst [hunk 3]
```diff
@@ -47,7 +46,8 @@ non-root user.
 signature travels with the container image, allowing you to verify that
 the image is unmodified at any time. Encryption of containers using
 LUKS2 is also supported. Encrypted containers can be run without
-decrypting them to disk first.
+decrypting them to disk first when using a setuid installation of
```
Comment (ikaneshiro) [inline index.rst:49]: I would be careful with wording here to make sure we don't imply that non-setuid installations decrypt content to disk first. Maybe:

Encrypted containers can be run without decrypting them to disk first. Note: Execution of encrypted containers currently requires a setuid installtion.

File: persistent_overlays.rst [hunk 4]
```diff
@@ -11,39 +11,29 @@ changes are kept separately from the base container image.
...
 container finishes executing.
 
-You can use persistent overlays with the following commands:
-
--  ``run``
--  ``exec``
--  ``shell``
--  ``instance.start``
+You can use persistent overlays with the following commands: ``run``,
+``exec``, ``shell``, and ``instance.start``.
```
Comment (ikaneshiro) [inline persistent_overlays.rst:31]:
```suggestion
``exec``, ``shell``, and ``instance start``.
```

File: persistent_overlays.rst [hunk 5]
```diff
@@ -133,20 +199,17 @@ Overlay embedded in SIF
...
+An example of doing that directly with the ``{command} overlay create``
+command was shown above,
+but an external image file can also be added to a SIF file with the
+``{command} sif add`` command like this:
 
 .. code::
 
-   $ {command} sif add --datatype 4 --partfs 2 --parttype 4 --partarch 2 --groupid 1 ubuntu_latest.sif overlay.img
+   $ {command} sif add --datatype 4 --partfs 2 --parttype 4 --partarch 2 --groupid 1 ubuntu.sif overlay.img
+   $ {command} sif list ubuntu.sif|grep -i ext3
```
Comment (ikaneshiro) [inline persistent_overlays.rst:211]:
```suggestion
   $ {command} sif list ubuntu.sif | grep -i ext3
```

**Review by ikaneshiro — COMMENTED** [review]
Vast majority of changes lgtm

File: definition_files.rst (hunk 2 above)
Comment (DrDaveD) [reply definition_files.rst:103]: I know. I had added the label at an artificial place, where the oci bundle was described in oci_runtime.rst, but that probably wasn't a good idea.  I took it out.

**Pull request merged**